build.yaml

```
@@ -1,8 +1,8 @@
 ---
 name: "jellyfin-plugin-pushbullet"
 guid: "de228f12-e43e-4bd9-9fc0-2830819c3b92"
-version: "3" # Please increment with each pull request
-jellyfin_version: "10.5.0" # The earliest binary-compatible version
+version: "3"
```

```suggestion
version: "4"
```

As version 3 was already released.